paymentsheet/src/test/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadataTest.kt

```diff
@@ -729,7 +728,7 @@ internal class PaymentMethodMetadataTest {
         assertThat(metadata.isExternalPaymentMethod("card")).isFalse()
     }
 
-    @OptIn(ExperimentalCardBrandFilteringApi::class)
+    @Suppress("LongMethod")
```

Comment:
Not sure why this is getting flagged now, it's 65 lines long, the test below it uses the @Supress so I think this is fine.